src/inverseRobot.js

```diff
@@ -24,7 +24,21 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  let result = [];
+  const values = Object.values(robot);
+  const check = new Set(values);
```

Comment:
what does check mean?
```suggestion
  const uniqueValues = new Set(values);
```